examples/ES/README.md

```diff
@@ -15,7 +15,7 @@ Please see [here](https://github.com/openai/mujoco-py) to know more about Mujoco
 ## How to use
 ### Dependencies
 + [paddlepaddle>=2.0.0](https://github.com/PaddlePaddle/Paddle)
-+ [parl>=1.4](https://github.com/PaddlePaddle/PARL)
++ [parl>=2.0.0](https://github.com/PaddlePaddle/PARL)
```

Comment:
parl>=2.0.2？

Reply:
ES does not develop the algorithm in `parl.algorithm`.